tools/update_requirements.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+import sys
+from pathlib import Path
+
+try:
+    import tomllib
+except ImportError:
+    import tomli as tomllib
```

Comment:
Preferring the builtin for recent Python.

Reply:
Nice!